modules/cnf-migrating-from-amqp-to-http-transport.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-migrating-from-amqp-to-http-transport_{context}"]
+= Migrating consumer applications to use HTTP transport for PTP and bare-metal events
+
+If you have previously deployed PTP or bare-metal events consumer applications, you need to update the applications to use HTTP message transport.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have updated the PTP Operator or {redfish-operator} to version 4.13+ which uses HTTP transport by default.
+
+.Procedure
+
+. Get the name of the PTP or bare-metal events `transportHost` in the cluster. For example:
+
+.. To get the name of the PTP events `transportHost`, run the following command:
++
+[source,terminal]
+----
+$ oc get ptpoperatorconfig -n openshift-ptp -o jsonpath='{ .items[*].spec.ptpEventConfig.transportHost }'
+----
++
+.Example output
+[source,terminal]
+----
+http://ptp-event-publisher-service-compute-1-example.openshift-ptp.svc.cluster.local:9043
+----
+
+.. To get the name of the bare-metal events `transportHost`, run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-bare-metal-events get hardwareevent -o jsonpath='{.items[*].spec.transportHost}{"\n"}'
+----
++
+.Example output
+[source,terminal]
+----
+http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043
+----
+
+. Update your events consumer application to use HTTP transport.
```

Comment:
We don't want to give our sample example to DU example. Probably we should have this in consumer implementation section

Reply:
OK, I've updated this a bit to remove the sample consumer detail. PTAL.